### ISSUE TITLE
feature request: setting the transfer of the cache to the memory card, like protox or telegram.

### ISSUE THREAD

**Issue by NektoNektovich** [evilcorpltd/aTox#1046]
feature request: setting the transfer of the cache to the memory card, like protox or telegram.

**Comment by robinlinden** [comment]
This is already supported using the standard [installLocation-system in Android](https://developer.android.com/guide/topics/data/install-location), so aTox can be moved to the SD card from the app settings.

What are you missing with this feature?

**Comment by NektoNektovich** [comment]
I have three android 6.0 and 7.1.1 phones, none of them have transfer
applications to a memory card, but I will not buy another phone, because
there is no model that I would like at least a little.

суббота, 5 ноября 2022 г. пользователь Robin Lindén <
***@***.***> написал:
> This is already supported using the standard installLocation-system in
Android, so aTox can be moved to the SD card from the app settings.
>
> What are you missing with this feature?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci4.googleusercontent.com/proxy/vz_jU04MyawREtjD8Mjk4VkZg2ucdd2vW6bXlhjJil0DLpp1x2uDSJlLkanl8OUrX9hC70tcipGGJ8FaYuaenCokFBbAVZdWjR8wqx6ziBV4Bh1hxKY0W4t01dMItSPv_74u2_mpnHaXL2as2HRD7Ww2Op3nRlTj9YYGGqW3PuOWy6OeW8F8wj8FL05j3CSkNHaX2qm4c30kL8XMYX5qqG3ucmWdTlI0khhUJkyJlg=s0-d-e1-ft#https://github.com/notifications/beacon/AMZDE6CA2YBS4F7NJGQXXN3WGZJZ7A5CNFSM5XOKGGQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJXA3OHI.gif>Message
ID: ***@***.***>

**Comment by theAkito** [comment]
> I have three android 6.0 and 7.1.1 phones, none of them have transfer applications to a memory card, but I will not buy another phone, because there is no model that I would like at least a little.

That is too old, anyway. If you would like to continue using Android, you should buy a phone with a newer Android version or one that supports a custom ROM with a newer version.

Otherwise, you would probably need to drop Android and resort to an alternative or just run plain Linux on a mobile device.

###### My 2 cents.

**Comment by NektoNektovich** [comment]
Are you planning on dropping support for Android 6 soon?😯

суббота, 5 ноября 2022 г. пользователь theAkito ***@***.***>
написал:
> I have three android 6.0 and 7.1.1 phones, none of them have transfer
applications to a memory card, but I will not buy another phone, because
there is no model that I would like at least a little.
>
> That is too old, anyway. If you would like to continue using Android, you
should buy a phone with a newer Android version or one that supports a
custom ROM with a newer version.
>
> Otherwise, you would probably need to drop Android and resort to an
alternative or just run plain Linux on a mobile device.
>
> My 2 cents.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci6.googleusercontent.com/proxy/g0JAu_NtAwaRvRfB_OGz8KOUOUTtOjdiMEsrFDKEwqgNFpnZIZQ0p_MBzO5_utkWeDb_0BW67t4oqU9AIp9QE2IbARjc9vO08OoKVYVcvkf1EpB9sghOXA4j9QRaYrZf752cMpizZftIW0LKLXBVKDrVbhoYx_ELuxkWrLMHkRC9xrVTd1sjUTdIpc4KWaPNgGhyUESdwDsjuEhwwigpVMmax5lJiOeH0RKz3r7rqQ=s0-d-e1-ft#https://github.com/notifications/beacon/AMZDE6CMCZGONNZ4XXZWFIDWGZ2L3A5CNFSM5XOKGGQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJXBBIBA.gif>Message
ID: ***@***.***>

**Comment by theAkito** [comment]
> Are you planning on dropping support for Android 6 soon?😯 суббота, 5 ноября 2022 г. пользователь theAkito ***@***.***> написал:
> I have three android 6.0 and 7.1.1 phones, none of them have transfer
> applications to a memory card, but I will not buy another phone, because there is no model that I would like at least a little.
> That is too old, anyway. If you would like to continue using Android, you
> should buy a phone with a newer Android version or one that supports a custom ROM with a newer version.
> Otherwise, you would probably need to drop Android and resort to an
> alternative or just run plain Linux on a mobile device.
> My 2 cents. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.<
> https://ci6.googleusercontent.com/proxy/g0JAu_NtAwaRvRfB_OGz8KOUOUTtOjdiMEsrFDKEwqgNFpnZIZQ0p_MBzO5_utkWeDb_0BW67t4oqU9AIp9QE2IbARjc9vO08OoKVYVcvkf1EpB9sghOXA4j9QRaYrZf752cMpizZftIW0LKLXBVKDrVbhoYx_ELuxkWrLMHkRC9xrVTd1sjUTdIpc4KWaPNgGhyUESdwDsjuEhwwigpVMmax5lJiOeH0RKz3r7rqQ=s0-d-e1-ft#https://github.com/notifications/beacon/AMZDE6CMCZGONNZ4XXZWFIDWGZ2L3A5CNFSM5XOKGGQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJXBBIBA.gif>Message ID: ***@***.***>

No, I'm not planning anything. I was just adding my personal view to the topic, which is not necessarily the view of the project.

**Comment by robinlinden** [comment]
@NektoNektovich Are you sure there's no option for it? What kind of phones are they? Even on Android 5.0 I can see that the option is there and works:
![image](https://user-images.githubusercontent.com/8304462/200131319-6c7f5184-60b3-4886-9168-908438cde19b.png)

I don't have anything older to test on right now, but 5.0 is older than what you have so I see no reason for why it wouldn't work.

And we're not dropping support for Android 4.4 any time soon.

**Comment by NektoNektovich** [comment]
yes, android 4.4 and 5 supports transferring the application to a memory
card, but for android 6 and 7, Google removed this feature, replacing it
with formatting the memory card as internal memory. but it doesn't work,
after rebooting the phone the memory card falls off and is not visible 🤷‍♂️

суббота, 5 ноября 2022 г. пользователь Robin Lindén <
***@***.***> написал:
> @NektoNektovich Are you sure there's no option for it? What kind of
phones are they? Even on Android 5.0 I can see that the option is there and
works:
> <
https://ci6.googleusercontent.com/proxy/xR60De0j9TnIx-XR5_NcJxn2nCKdI7lWEEjw1wXGPLIwanhIJzw-mFcL5V5-g_3JjXvPPjvg6z38w1pSCMluCovRDQhTmHPnX4Mh_vPYQ8JbpB8SySoHgud0BYjNKVwr3y6b0l_NcyXYdWMwWster0RfsH7nGNs=s0-d-e1-ft#https://user-images.githubusercontent.com/8304462/200131319-6c7f5184-60b3-4886-9168-908438cde19b.png
>
>
> I don't have anything older to test on right now, but 5.0 is older than
what you have so I see no reason for why it wouldn't work.
>
> And we're not dropping support for Android 4.4 any time soon.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.<
https://ci4.googleusercontent.com/proxy/pQzfwcwCQNFA6YRJka9GQoQUbfD0WZLELHYWAtv4c9299YFK3f7inuAuew1y_cYZWjqeCgRCgsEUG6SpHcGl7z-F9lpHCezWNfYdSsnXNVM0vYdQ2ij155rtzSJsidAwVsJPDWb1dPU2Ggb7O9T1w3vkyoUDeMbQJMJUM7EDRgTZe_aprNm62689kazvtFjlOo_ETzL_6zSgvpa_6GZjBn21WiomKdno_qAw6-rekw=s0-d-e1-ft#https://github.com/notifications/beacon/AMZDE6F4KUKAUQC5KADIXYDWG2FAHA5CNFSM5XOKGGQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJXBF4XA.gif>Message
ID: ***@***.***>

**Comment by theAkito** [comment]
> but it doesn't work, after rebooting the phone the memory card falls off and is not visible 🤷‍♂️

This is the intended behaviour. Formatting it as "internal memory" means, it extends the internal memory by the SD card. Then, it's not a "classical" SD card anymore, as you would expect. It's kind of "merged" with the internal memory.
The point is, that using an SD card suddenly becomes seamless, but you have to pay the price of not being able to just remove the SD card & plug it in to the computer just like that, anymore.

**Comment by NektoNektovich** [comment]
I know all this👌 but the function does not work, after rebooting the
phone, the memory card is not mounted, the phone does not see it. so my
memory cards are formatted as removable.

суббота, 5 ноября 2022 г. пользователь theAkito ***@***.***>
написал:
> but it doesn't work, after rebooting the phone the memory card falls off
and is not visible 🤷‍♂️
>
> This is the intended behaviour. Formatting it as "internal memory" means,
it extends the internal memory by the SD card. Then, it's not a "classical"
SD card anymore, as you would expect. It's kind of "merged" with the
internal memory.
> The point is, that using an SD card suddenly becomes seamless, but you
have to pay the price of not being able to just remove the SD card & plug
it in to the computer just like that, anymore.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.<
https://ci5.googleusercontent.com/proxy/OvZ0sUDV5C-wIQRpT_wkOtil1VplepYHkMMZyoHJAWudJYCDmQRHynjCjakpi3RPOXqqZxNWtWzJN3XK-hLXMSdWcPg2oBKFcp3zbYCz2hFtOl8YBXIyDu47xupTP4UyDh9h93swHCKHzzBXZx17yZTxEDHURFqvpfRWXAWJIC0S5nXIOWZ1HjbBdLhFK7oehlbBK3Ph4rcQ1kkGTgcKRRMLRodYjyg4cl6Uke6-8Q=s0-d-e1-ft#https://github.com/notifications/beacon/AMZDE6BZMOXFFSLBEMZUX3TWG2ICDA5CNFSM5XOKGGQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOJXBHY4I.gif>Message
ID: ***@***.***>
